hudi-utilities/src/main/java/org/apache/hudi/utilities/streamer/HoodieStreamer.java

```diff
@@ -163,6 +163,9 @@ public HoodieStreamer(Config cfg, JavaSparkContext jssc, FileSystem fs, Configur
     cfg.recordMergeMode = mergingConfigs.getLeft();
     cfg.payloadClassName = mergingConfigs.getMiddle();
     cfg.recordMergeStrategyId = mergingConfigs.getRight();
+    this.properties.put(HoodieTableConfig.RECORD_MERGE_MODE.key(), cfg.recordMergeMode.name());
```

Comment:
@nsivabalan from my understanding, the RECORD_MERGE_STRATEGY_ID and mode should always be consistent.  Do we have a case where mode stays the same but the strategy id gets changed? (just wondering if just use mode / id would be sufficient for this check.

Reply:
no worries. i've figured out how to change the id.